commercial-schedules/src/main/java/org/dcsa/conformance/standards/cs/CsScenarioListBuilder.java

```diff
@@ -10,175 +10,380 @@
 import lombok.extern.slf4j.Slf4j;
 import org.dcsa.conformance.core.scenario.ConformanceAction;
 import org.dcsa.conformance.core.scenario.ScenarioListBuilder;
-import org.dcsa.conformance.standards.cs.action.CsGetPortSchedulesAction;
-import org.dcsa.conformance.standards.cs.action.CsGetRoutingsAction;
-import org.dcsa.conformance.standards.cs.action.CsGetVesselSchedulesAction;
-import org.dcsa.conformance.standards.cs.action.SupplyScenarioParametersAction;
+import org.dcsa.conformance.standards.cs.action.*;
```

Comment:
We should avoid using star imports. Can you change that setting in your IDE? I tried to enforce it as well with settings in the project, but somehow it does not work...

Reply:
I did change that in the settings but it didn't apply to the already existing * imports (even after forcefully doing organize imports) so I manually had to do.

Reply:
Ah okay.. sometimes I delete the .* lines and press Organize Imports, and it got fixed. I only made a remark about it, because the PR was grouping them.. otherwise, I'd let it stay. Hopefully your IDE will now automatically chose the right imports. Thanks